### PR TITLE
Exercise4 driver finder

### PR DESCRIPTION
**Pull request by MarStr12**

### CODE REVIEW

File: 4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinderNEQL.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    private void processEntity(Entity entity) {
+        BComponent component = (BComponent)entity;
+        logger.info("Is not in Driver Container: " + component.getSlotPath());
+    }
+    @Override
+    public void started() { executor = newSingleThreadExecutor();}
+    @Override
+    public void stopped() {
+        if (executor != null) {
+            executor.shutdownNow();
```
Comment (robert-tridium) [inline 4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinderNEQL.java:72]: This line will cause an AccessControlException unless you modify your module-permissions.xml file the way we did in Lab 5.

File: 4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinder.java
Comment (robert-tridium) [inline 4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinder.java:1]: I like how in this class and in `BDriverFinderNEQL` you're experimenting with different kinds of multithreading.

File: 4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinderNEQL.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  public Type getType() { return TYPE; }
+  public static final Type TYPE = Sys.loadType(BDriverFinderNEQL.class);
+
+  //endregion Type
+
+//@formatter:on
+//endregion /*+ ------------ END BAJA AUTO GENERATED CODE -------------- +*/
+
+    public void doFind (Context context) {
+        BQueryResult result = (BQueryResult)BOrd.make("neql:n:network and n:parent->n:type!='driver:DriverContainer'").get(Sys.getStation(), context);
```
Comment (robert-tridium) [inline 4 Ord Queries Exercise - Driver Finder/driverFinder-rt/src/com/tridiumuniversity/driverFinder/BDriverFinderNEQL.java:60]: Query looks great.

**Comment by MarStr12** [comment]
Thanks for reviewing!

**Pull request closed without merging**